exo-deploy/src/deployers/aws-deployer.ls

```diff
@@ -54,7 +54,7 @@ class AwsDeployer
 
   teardown: ({nuke}, done) ->
     @terraform-file-builder.generate-provider-credentials!
-    process.stdout.write "terraform starting nuke from AWS"
+    process.stdout.write "terraform starting teardown from AWS"
```

Comment:
Well, it's actually both. If `nuke` is set in the parameters, it's teardown of application infrastructure including the hosted zone. If not, it's teardown from AWS. That distinction is already made in earlier print statements, this statement is just supposed to say "we're starting teardown now".